### PR TITLE
Fix Flow violation

### PR DESCRIPTION
**Pull request by gaearon**
It got broken by #8855.

### CODE REVIEW

File: src/shared/utils/getComponentName.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function getComponentName(instanceOrFiber : ReactInstance | Fiber) : string | nu
   if (typeof instanceOrFiber.getName === 'function') {
     // Stack reconciler
     const instance = ((instanceOrFiber : any) : ReactInstance);
-    return instance.getName() || 'Component';
+    return instance.getName() || 'Component' || null;
```
Comment (edvinerikson) [inline src/shared/utils/getComponentName.js]: is this one necessary?

Comment (gaearon) [inline src/shared/utils/getComponentName.js]: Oops no. Send a pr?

Comment (edvinerikson) [inline src/shared/utils/getComponentName.js]: sure 👍

Comment (edvinerikson) [inline src/shared/utils/getComponentName.js]: https://github.com/facebook/react/pull/8858

**Comment by spicyj** [comment]
Thank you.

**Pull request merged**